admin_guide/managing_networking.adoc

```diff
@@ -86,19 +86,40 @@ Alternatively, instead of specifying specific project names, you can use the
 `--selector=<project_selector>` option.
 
 [[admin-guide-disabling-hostname-collision]]
-== Disabling Host Name Collision Prevention For Ingress Objects
+== Disabling Host Name Collision Prevention For Routes and Ingress Objects
 
 In {product-title}, host name collision prevention for routes and ingress
-objects is enabled by default. This means that the host name in a route or
-ingress object can only be set on creation and not edited afterwards. Disabling
-host name collision prevention lets you edit a host name for ingress objects after creation.
-However, because {product-title} uses the object creation timestamp to determine
-the oldest route or ingress object for a given host name, the route or ingress
-object can hijack a host name with a newer route. This can happen if an older
-route changes its host name, or if an ingress object is introduced.
-
-This is relevant to {product-title} installations that depend upon Kubernetes
-behavior, including allowing the host names in ingress objects be edited.
+objects is enabled by default. This means that regular users can set the host
+name in a route or ingress object only on creation and cannot change it
+afterwards.  However, you can relax this restriction on routes and ingress
+objects for some or all users.
+
+[CAUTION]
```

Comment:
s/CAUTION/WARNING

I don't think we use caution anywhere.

Reply:
Thanks! I used "CAUTION" because I saw it further down in `managing_networking.adoc`, but "WARNING" does seem to be much more common in other files.  Fixed.